tests/BenchmarkDotNet.IntegrationTests/ParamSourceTests.cs

```diff
@@ -35,5 +50,159 @@ public IEnumerable<string> StringValues
             [Benchmark]
             public void Method() { }
         }
+
+        private Summary CanExecuteWithExtraInfo(Type type, IToolchain toolchain)
+        {
+            IConfig config = CreateSimpleConfig(job: Job.Dry.WithToolchain(toolchain));
+            // Show the relevant codegen excerpt in test results (the *.notcs is not part of the logs)
+            // This uses the private static CodeGenerator.GetParamsContent() method.
+            // Method changes there must be applied here - or consider making the method internal.
+            // TODO Should we omit the extra info if (toolchain.IsInProcess)? Doesn't use GetParamsContent().
+            Output.WriteLine("// Benchmarks and CodeGenerator.GetParamsContent()");
+            BenchmarkRunInfo runInfo = BenchmarkConverter.TypeToBenchmarks(type, config);
+            MethodInfo getParamsContent = typeof(Code.CodeGenerator).GetMethod("GetParamsContent", BindingFlags.Static | BindingFlags.NonPublic, null, new[] { typeof(BenchmarkCase) }, null);
```

Comment:
I added a bit of a hack to add extra info to the test results. If that's frowned upon, I can remove it.